cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,46 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
 
-describe('Settings page', () => {
-  before(() => {
+import SettingsPageObject from '../support/pages/settings.pageObject';
 
-  });
-
-  beforeEach(() => {
+beforeEach(() => {
+  const settingsPage = new SettingsPageObject();
 
+  describe('Settings page', () => {
+    cy.task('db:clear');
   });
 
   it('should provide an ability to update username', () => {
+    settingsPage.visit();
+    settingsPage.updateUsername('NewUsername');
 
+    cy.contains('NewUsername').should('be.visible');
   });
 
   it('should provide an ability to update bio', () => {
+    settingsPage.visit();
+    settingsPage.updateBio('This is my updated bio');
 
+    cy.contains('This is my updated bio').should('be.visible');
   });
 
   it('should provide an ability to update an email', () => {
+    settingsPage.visit();
+    settingsPage.updateEmail('newemail@example.com');
 
+    cy.contains('Your email has been updated').should('be.visible');
   });
 
   it('should provide an ability to update password', () => {
+    settingsPage.visit();
```

Comment:
Verify that the `visit` method is correctly implemented in the `SettingsPageObject` class. This method should navigate to the settings page.